app/workers/data_update_worker.rb

```diff
@@ -31,7 +31,7 @@ def log_status(script)
     status = script.status
     file_name = script.file_name
 
-    logger_destination = status == :failed ? :error : :info
+    logger_destination = status.to_sym == :failed ? :error : :info
```

Comment:
Whilst I was testing the failed case I was getting a string "failed" and so I just added .to_sym as a protection in the case that this happens.